nextcord/ui/select/channel.py

```diff
@@ -0,0 +1,246 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-present Rapptz
+Copyright (c) 2021-present tag-epic
```

Comment:
```suggestion
Copyright (c) 2022-present tag-epic
```

All of this code *was* written by you, right?